config/deploy.rb

```diff
@@ -6,10 +6,10 @@
 
 set :home_directory, "/home/lyberadmin"
 
-set :branch, 'master'
+#set :branch, 'master'
```

Comment:
i've used this pattern in other projects since it allows you to deploy a different branch other than master, which can be very useful (particularly in -dev and -stage).  In fact I did just that when testing my work in this branch.  We could make the default master instead of the last branch pushed, so you can just push return when it asks instead of entering it.